chapter13/src/main/scala/chapter13/ActiveActive.scala

```diff
@@ -92,7 +92,7 @@ object ActiveActive {
         if (answer.isDefined) {
           val right = answer.get
           val wrong = replies.collect {
-            case SeqResult(_, res, replica, _) if res != right ⇒ replica
+            case SeqResult(_, result, replica, _) if res != right ⇒ replica
```

Comment:
The `res` shallows the outside `res` which will always be null, then will never become to `Known` state from `Unknown`.